generate.sh

```diff
@@ -27,20 +34,19 @@ if [ "${WEBUI:-false}" == "true" ]; then
   else
     mkdir -p temp/${EMAIL} && cd $_
     echo -e "Cloning Repository...\n"
-    git clone -q ${GITURL} "$UNIQUEID" && cd $_
+    git clone -q https://:@${PLATFORM}/${USERNAME}/${REPONAME} "$UNIQUEID" >/dev/null 2>&1
+    if [ $? -ne 0 ]; then
+      >&2 echo "Failed to Clone. Repository does not exist."
```

Comment:
add a `\n` at the end of the message? would be consistent with other error messages.